epochStart/metachain/rewards.go

```diff
@@ -41,7 +41,7 @@ type ArgsNewRewardsCreator struct {
 	NodesConfigProvider           epochStart.NodesConfigProvider
 	// this fixes the situation where a node has 0 proposed blocks and 0 validator failures
 	// but does not receive rewards.
-	RewardsFix1EpochEnable        uint32
+	RewardsFix1EpochEnable uint32
```

Comment:
where do we use this field?

Reply:
it is set to the struct member `rewardsFix1EnableEpoch` and is used here:
```
func (rc *rewardsCreator) isRewardsFix1Enabled(epoch uint32) bool {
	return epoch > rc.rewardsFix1EnableEpoch
}
```
this function is called and proceed accrodingly in `computeValidatorInfoPerRewardAddress`